src/components/NavigationBar.jsx

```diff
@@ -0,0 +1,18 @@
+import Container from 'react-bootstrap/Container';
+import Nav from 'react-bootstrap/Nav';
+import Navbar from 'react-bootstrap/Navbar';
+
+const NavigationBar = () => {
+    return (
+    <Navbar bg="dark" variant="dark">
+      <Navbar.Brand>NC-News</Navbar.Brand>
+      <Container>
+      <Nav className="me-auto">
+        <Nav.Link href="/">All articles</Nav.Link>
```

Comment:
You should use a `Link` from `react-router-dom` here instead of a `Nav.Link` from `react-bootstrap`. The bootstrap link is an a tag that will do a hard refresh of the page which is something you don't want to happen.